### PR TITLE
 BrenKt-SK963-TRANS-WEB-Email-alloffers-redirection_alias

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web/emails/feature_redirections/guide.es-es.md [hunk 1]
```diff
@@ -1,128 +1,192 @@
...
 
-#### Paso 1: Acceder a la gestión de las redirecciones
-Por defecto, estará en la pestaña de `Información general`{.action} de su MX Plan. Haga clic sobre la pestaña `Correo electrónico`{.action} y a continuación en el botón de la derecha `Gestión de las redirecciones`{.action}.
+## Procedimiento
 
-![correo electrónico](images/mxplan-legacy-1.png){.thumbnail}
+Los capítulos "[crear una redirección](#redirect)" y "[crear un alias](#alias)" se refieren al conjunto de nuestros productos de correo de OVHcloud, **excepto si**:
 
+- Si tiene la versión histórica de la solución MXplan, puede comprobarla en el siguiente cuadro.
+- si tiene un dominio registrado con OVHcloud y no tiene ningún servicio de correo asociado,
```
Comment (OVH-Team-Guides) [inline pages/web/emails/feature_redirections/guide.es-es.md]:
```suggestion
- Si tiene un dominio registrado con OVHcloud y no tiene ningún servicio de correo asociado.
```

File: pages/web/emails/feature_redirections/guide.es-es.md [hunk 2]
```diff
@@ -1,128 +1,192 @@
...
+- Haber iniciado sesión en el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es).
+- Disponer de una solución de correo electrónico de OVHcloud previamente configurada (**MX Plan**, incluida en nuestros [planes de hosting](https://www.ovhcloud.com/es-es/web-hosting/), incluida en un [alojamiento Start10M gratuito](https://www.ovhcloud.com/es-es/domains/free-web-hosting/) o contratada por separado como solución autónoma, como [Hosted Exchange](https://www.ovhcloud.com/es-es/emails/hosted-exchange/) o [Email Pro](https://www.ovhcloud.com/es-es/emails/email-pro/)).
 
-#### Paso 1: Acceder a la gestión de las redirecciones
-Por defecto, estará en la pestaña de `Información general`{.action} de su MX Plan. Haga clic sobre la pestaña `Correo electrónico`{.action} y a continuación en el botón de la derecha `Gestión de las redirecciones`{.action}.
+## Procedimiento
 
-![correo electrónico](images/mxplan-legacy-1.png){.thumbnail}
+Los capítulos "[crear una redirección](#redirect)" y "[crear un alias](#alias)" se refieren al conjunto de nuestros productos de correo de OVHcloud, **excepto si**:
```
Comment (OVH-Team-Guides) [inline pages/web/emails/feature_redirections/guide.es-es.md]:
```suggestion
Los capítulos "[crear una redirección](#redirect)" y "[crear un alias](#alias)" se refieren al conjunto de nuestros productos de correo de OVHcloud, **excepto**:
```

File: pages/web/emails/feature_redirections/guide.es-es.md [hunk 3]
```diff
@@ -1,128 +1,192 @@
...
+>
+> En función de la fecha de activación de su MXplan o si [esta ha sido migrada recientemente](https://www.ovhcloud.com/es-es/web-hosting/mxplan-migration/), es posible que disponga de la versión histórica o de la nueva versión del producto. Antes de continuar, es necesario identificarla.<br>
+>
+> Para ello, conéctese a su [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es), en la sección `Web Cloud`{.action}. Haga clic en `Correo electrónico`{.action} y seleccione el servicio MX Plan correspondiente. Siga leyendo esta guía en función de su versión:<br>
+>
+> |Versión histórica de la solución MX Plan|Nueva versión de la solución MX Plan|
+> |---|---|
+> |![email](images/mxplan-starter-legacy.png){.thumbnail}<br> Encontrar la oferta en el recuadro "Suscripción"|![email](images/mxplan-starter-new.png){.thumbnail}<br>Consulte la `referencia del servidor` en el recuadro "Resumen"|
+> |Continúe leyendo esta guía en el apartado « [Versión histórica de la solución MX Plan](#mxplanlegacy) ».|Continúe leyendo esta guía en el apartado [Crear una redirección](#redirect) o [Crear un alias.](#alias)|
```
Comment (OVH-Team-Guides) [inline pages/web/emails/feature_redirections/guide.es-es.md]:
```suggestion
> |Continúe leyendo esta guía en el apartado "[Versión histórica de la solución MX Plan](#mxplanlegacy)".|Continúe leyendo esta guía en el apartado [Crear una redirección](#redirect) o [Crear un alias.](#alias)|
```

File: pages/web/emails/feature_redirections/guide.es-es.md [hunk 4]
```diff
@@ -1,128 +1,192 @@
...
-Haga clic en `Aceptar`{.action} para confirmar la redirección añadida.
+![correo electrónico](images/webmail.png){.thumbnail}
 
-![correo electrónico](images/mxplan-legacy-3.png){.thumbnail}
+En nuestro ejemplo, se muestra una **redirección con copia local** (véase el [esquema 2](#diagram) al inicio de esta guía). Si esto es lo que necesita, haga clic en `OK`{.action} (con el icono de disquete) en la parte superior izquierda y se aplicará la regla. De lo contrario, vaya al paso siguiente.
+
+Siga los pasos que se describen haciendo clic en cada pestaña:
+
+> [!tabs]
+> **Etape 1**
```
Comment (OVH-Team-Guides) [inline pages/web/emails/feature_redirections/guide.es-es.md]: Please replace Etape with Etapa for all steps below

File: pages/web/emails/feature_redirections/guide.it-it.md [hunk 5]
```diff
@@ -1,128 +1,195 @@
...
-Clicca su `Conferma`{.action} per aggiungere il reindirizzamento.
+![email](images/webmail.png){.thumbnail}
 
-![email](images/mxplan-legacy-3.png){.thumbnail}
+Nel nostro esempio, si tratta di un **reindirizzamento con copia in locale** (vedi lo [schema 2](#diagram) all’inizio di questa guida). Se questo è ciò di cui hai bisogno, clicca su `OK`{.action} (icona del floppy disk) in alto a sinistra e la regola diventa effettiva. Altrimenti, passa allo step successivo.
+
+Per seguire gli step descritti, clicca su ciascuna scheda qui sotto:
+
+> [!tabs]
+> **Etape 1**
```
Comment (OVH-Team-Guides) [inline pages/web/emails/feature_redirections/guide.it-it.md]: Etape to translate (same for other languages)

File: pages/web/emails/feature_redirections/guide.it-it.md [hunk 6]
```diff
@@ -1,128 +1,195 @@
...
+>>
+> **Etape 3**
+>>
+>> Dalla pagina di gestione **Regole di Posta in arrivo**, clicca sul simbolo`+`{.action} in alto a sinistra.<br><br>
+>> ![email](images/emails-all-03.png){.thumbnail}<br><br>
+>>
+> **Etape 4**
+>>
+>> **Nome**: definisci il nome del tuo reindirizzamento. <br>
+>> **Quando il messaggio arriva e soddisfa tutte queste condizioni** : se il tuo reindirizzamento si applica a tutti i messaggi, seleziona `[Applica a tutti i messaggi]`{.action}.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/feature_redirections/guide.it-it.md:111]: Action format does not need []

File: pages/web/emails/feature_redirections/guide.pl-pl.md [hunk 7]
```diff
@@ -1,130 +1,197 @@
...
+>>
+> **Etap 3**
+>>
+>> W oknie zarządzania **Reguły skrzynki odbiorczej** w lewym górnym rogu kliknij ikonę `+`{.action}.<br><br>
+>> ![emails](images/emails-all-03.png){.thumbnail}<br><br>
+>>
+> **Etap 4**
+>>
+>> **Nazwa**: zdefiniuj nazwę przekierowania. <br>
+>> **Kiedy wiadomość przychodzi i spełnia wszystkie te warunki**: jeśli przekierowanie dotyczy wszystkich wiadomości, wybierz `[Zastosuj do wszystkich wiadomości]`{.action}.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/feature_redirections/guide.pl-pl.md:111]: Action format to correct (same on other languages)

File: pages/web/emails/feature_redirections/guide.pl-pl.md [hunk 8]
```diff
@@ -1,130 +1,197 @@
...
+>>![emails](images/emails-all-04.png){.thumbnail .w-640}<br><br>
+>>
+> **Etap 5**
+>>
+>> **Wykonać wszystkie następujące** czynności: w tym przypadku stosujesz przekierowanie, wybierz pozycję `Przekaż, przekieruj lub wyślij`{.action}, a następnie `Przekieruj wiadomość do...`{.action}.<br><br>
+>>![emails](images/emails-all-05.png){.thumbnail .w-640}<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/feature_redirections/guide.pl-pl.md:117]: why the " .w640" ?

File: pages/web/emails/feature_redirections/guide.pl-pl.md [hunk 9]
```diff
@@ -1,130 +1,197 @@
...
+> [!primary]
+>
+> Jeśli wybierzesz tryb kopii "**Zachowaj kopię e-maila w OVHcloud**", na liście przekierowań automatycznie tworzone jest przekierowanie z adresu e-mail na ten sam adres, tworząc jego kopię lokalną.
+> 
 
 ## Sprawdź również
 
-Dołącz do społeczności naszych użytkowników na stronie <https://community.ovh.com/en/>.
\ No newline at end of file
+Dołącz do społeczności naszych użytkowników na stronie<https://community.ovh.com>.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/feature_redirections/guide.pl-pl.md]: Link should end with /en/
please fix and check other languages

File: pages/web/emails/feature_redirections/guide.pt-pt.md [hunk 10]
```diff
@@ -1,129 +1,197 @@
...
 
-![emails](images/mxplan-legacy-3.png){.thumbnail}
+Neste exemplo, trata-se de um **reencaminhamento com cópia local** (ver o [esquema 2](#diagram) no início deste guia). Se é o que deseja, clique em `OK`{.action} (ícone de disquete em cima, à esquerda) para a regra ser aplicada. Caso contrário, siga as instruções seguintes.
+
+Siga os passos descritos clicando sucessivamente em cada separador abaixo:
+
+> [!tabs]
+> **Etapa 1**
+>>
+>> Uma vez ligado ao seu endereço de e-mail através de [webmail](https://www.ovhcloud.com/pt/mail/), clique na roda dentada no canto superior direito e, a seguir, em `Options`{.action}.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/feature_redirections/guide.pt-pt.md]: Should "Options" be translated ?

File: pages/web/emails/feature_redirections/guide.pt-pt.md [hunk 11]
```diff
@@ -1,129 +1,197 @@
...
-> materializando a cópia local.
-> 
+> Para aplicar um **reencaminhamento simples** (ver [esquema 1](#diagram) no início deste manual), adicione uma regra suplementar ao seu **reencaminhamento com cópia local** a partir desta janela. Clique em `Adicionar uma ação`{.action} (quadro 1), depois em `Migrar, copiar ou suprimir`{.action} e, por fim, clique em `suprimir a mensagem`{.action}. Esta regra envia os e-mails diretamente para o lixo depois de ter reencaminhado as mensagens para o endereço final.<br><br>
+> ![e-mails](images/emails-all-07.png){.thumbnail .w-640}
 
-### Nova versão da oferta MX Plan
+### Criar um alias <a name="alias"></a>
 
-Na nova versão da oferta MX Plan, a gestão dos reencaminhamentos não é feita através da Área de Cliente, mas diretamente a partir do webmail do endereço em causa.
+Aceda à Área de [Cliente OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pt/&ovhSubsidiary=pt) e dirija-se à secção `Web Cloud`. De seguida, selecione o menu em função da sua oferta de e-mail:
```
Comment (OVH-Team-Guides) [inline pages/web/emails/feature_redirections/guide.pt-pt.md]:
```suggestion
Aceda à [Área de Cliente OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pt/&ovhSubsidiary=pt) e dirija-se à secção `Web Cloud`{.action}. De seguida, selecione o menu em função da sua oferta de e-mail:
```

Comment (OVH-Team-Guides) [reply pages/web/emails/feature_redirections/guide.pt-pt.md]: Please fix link on other occurrences in this version

File: pages/web/emails/feature_redirections/guide.pt-pt.md [hunk 12]
```diff
@@ -1,129 +1,197 @@
...
 
-![emails](images/mxplan-new-5.png){.thumbnail}
-
-Concluída a operação, clique em `OK`{.action} (ícone de disquete em cima, à esquerda).
+> [!primary]
+> Quando escolhe o modo de cópia "**Conservar uma cópia do e-mail na OVHcloud**", um reencaminhamento do endereço de e-mail para si próprio é automaticamente criado na lista dos reencaminhamentos, materializa esta cópia local.
+> 
 
-## Quer saber mais?
+## Saiba mais
```
Comment (OVH-Team-Guides) [inline pages/web/emails/feature_redirections/guide.pt-pt.md]: Please revert here. Check our templates.

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**